components/layout/product-grid-items.tsx

```diff
@@ -0,0 +1,29 @@
+import Grid from 'components/grid';
+import { GridTileImage } from 'components/grid/tile';
+import { Product } from 'lib/shopify/types';
+import Link from 'next/link';
+
+export default async function ProductGridItems({ products }: { products: Product[] }) {
```

Comment:
Should this be an async function?

Reply:
Uhh... No. There is no reason for it to be async.

Reply:
Addressed in #977.